README.md

```diff
@@ -105,7 +105,7 @@ sudo chown -R pi /srv
 Now copy configuration files from this project onto the Raspberry Pi:
 
 ```
-scp -r . pi@raspberrypi.local://srv
+rsync -e ssh -avz --progress --delete --exclude=.git . pi@raspberrypi.local://srv
```

Comment:
It depends on the client you're using. I left it there for compatibility reasons and in case user wants to pass additional parameters to ssh (like using another privkey)